check.js

```diff
@@ -0,0 +1,40 @@
+const check = (boxEle) => {
+  // change the ui to mark the task as complete
+  const buttons = boxEle.parentNode.lastElementChild;
+  const task = (boxEle.nextElementSibling);
+  buttons.classList.add('none');
+  task.classList.add('completed');
+};
+
+const unchecked = (boxEle) => {
+  const buttons = boxEle.parentNode.lastElementChild;
+  const task = (boxEle.nextElementSibling);
+  buttons.classList.remove('none');
+  task.classList.remove('completed');
+};
+
+const editVal = (index) => {
+  const arr = JSON.parse(localStorage.getItem('tasks'));
+  arr.forEach((task) => {
+    // eslint-disable-next-line eqeqeq
```

Comment:
[Optional] Kindly remove all disabled Linters, as it helps in writing more professional code

Reply:
I appreciate your feedback, the problem was addressed and solved.
Thank you.